recipes/arrow/all/conanfile.py

```diff
@@ -348,11 +215,28 @@ def requirements(self):
             self.requires("libbacktrace/cci.20210118")
 
     def validate(self):
-        # validate options with 'auto' as default value
-        auto_options = ["parquet", "compute", "dataset_modules", "with_boost", "with_flight_rpc", "with_gflags", "with_glog",
-                        "with_grpc", "with_jemalloc", "with_thrift", "with_llvm", "with_openssl", "with_protobuf", "with_re2", "with_utf8proc"]
-        for option in auto_options:
-            assert "auto" not in str(self.options.get_safe(option)), f"Option '{option}' contains 'auto' value, wich is not allowed. Generally the final value should be True/False"
+        # FIXME: Temporaly discard these configurations
+        if self.settings.compiler == "apple-clang":
+            raise ConanInvalidConfiguration("Not supported")
```

Comment:
```suggestion
        # FIXME: Temporaly discard these configurations
        if self.settings.compiler == "apple-clang":
            raise ConanInvalidConfiguration("Not supported")
```

It's not clear why this limitation now. Please, update it in the PR description if possible, and update the recipe with a better description.

Reply:
you are right, I added this to test some changes. I will remove this. Thanks! 🙏

Reply:
```suggestion
```